src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -2060,6 +2060,7 @@ shared_ptr<runtime::Executable>
         case OP_TYPEID::ScatterNDAdd:
         case OP_TYPEID::ShapeOf:
         case OP_TYPEID::SpaceToDepth:
+        case OP_TYPEID::SquaredDifference:
```

Comment:
As @dmyershov is pointing out on other PRs you may now try to add this op to implemented fused operations on IGPU (similar to this line: `"case OP_TYPEID::SquaredDifference:" line into switch of runtime::intelgpu::IntelGPUBackend::is_supported_impl function (intelgpu_backend.cpp).`

Reply:
Please check again